src/PrestaShopBundle/Command/AboutCommand.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Command;
+
+use PrestaShop\PrestaShop\Adapter\Configuration;
+use Symfony\Bundle\FrameworkBundle\Command\AboutCommand as BaseAboutCommand;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Helper\TableSeparator;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * This command displays information about the current PrestaShop installation.
+ * It extends the default Symfony about command to include project-specific details.
+ */
+#[AsCommand(name: 'about', description: 'Display information about the current project')]
+class AboutCommand extends BaseAboutCommand
+{
+    private $configuration;
+
+    public function __construct(Configuration $configuration)
+    {
+        parent::__construct();
+        $this->configuration = $configuration;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        parent::execute($input, $output);
+
+        $io = new SymfonyStyle($input, $output);
+
+        $rows = [
+            ['<info>Prestashop</>'],
+            new TableSeparator(),
+            ['Version', _PS_VERSION_],
+            ['Debug mode', _PS_MODE_DEV_ ? 'true' : 'false'],
+            ['Cache', $this->configuration->get('PS_SMARTY_CACHE') ? 'true' : 'false'],
```

Comment:
```suggestion
            ['Smarty Cache', $this->configuration->get('PS_SMARTY_CACHE') ? 'true' : 'false'],
```

Remember that there's a Smarty cache and an actual Cache section, where you can choose between Memcache, etc.

Reply:
You're absolutely right!